packages/notifications-toast/src/NotificationsToast.stylesheet.js

```diff
@@ -53,7 +53,13 @@ export default function stylesheet(props, themeData) {
       flexGrow: 1,
       display: `flex`,
       position: `relative`,
-      paddingRight: `36px`
+      paddingRight: `36px`, 
+      overflowWrap: 'break-word',
+      // flexDirection: 'row',
```

Comment:
Shouldn't have unused styles commented out. There's also an extra comma at the end of the previous line that should be removed:
`overflowWrap: 'break-word',`